policyengine_canada/variables/household/expenses/housing/public_or_non_profit_long_term_care_home.py

```diff
@@ -0,0 +1,9 @@
+from policyengine_canada.model_api import *
+
+
+class public_or_non_profit_long_term_care_home(Variable):
```

Comment:
Alternatively could we multiply rent by a bool indicating if this is where they lived?
```suggestion
class rent_paid_to_public_or_non_profit_long_term_care_home(Variable):
```